Sources/DangerFixtures/DangerFixtures.swift

```diff
@@ -29,6 +29,8 @@ public let githubEnterpriseFixtureDSL = parseDangerDSL(with: DSLGitHubEnterprise
 public let bitbucketFixtureDSL = parseDangerDSL(with: DSLBitBucketServerJSON)
 /// An example DSL using GitLab
 public let gitlabFixtureDSL = parseDangerDSL(with: DSLGitLabJSON)
+/// An example DSL without milestone date range using GitLab
+public let gitlabWithoutMilestoneDateRangeFixtureDSL = parseDangerDSL(with: DSLGitLabWithoutMilestoneDateRangeJSON)
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerFixtures/DangerFixtures.swift;
  Line: 33;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'gitlabWithoutMilestoneDateRangeFixtureDSL' (`identifier_name`)